### PR TITLE
M0228J-105 Release MicroJS 0.4.0

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: ApplicationDeveloperGuide/mmm.rst [hunk 1]
```diff
@@ -431,6 +432,7 @@ In order to use the MMM CLI for your project:
 where ``subcommand`` is the subcommand to execute (for example ``mmm build``).
 The available subcommands are:
 
+- init : create a project
```
Comment (bguedas) [inline ApplicationDeveloperGuide/mmm.rst:435]:
```suggestion
- ``init``: create a project
```

File: ApplicationDeveloperGuide/mmm.rst [hunk 2]
```diff
@@ -431,6 +432,7 @@ In order to use the MMM CLI for your project:
...
 - build : build the project
 - publish : publish the project
```
Comment (bguedas) [inline ApplicationDeveloperGuide/mmm.rst:437]:
```suggestion
- ``publish``: publish the project
```

File: ApplicationDeveloperGuide/mmm.rst [hunk 3]
```diff
@@ -431,6 +432,7 @@ In order to use the MMM CLI for your project:
 where ``subcommand`` is the subcommand to execute (for example ``mmm build``).
 The available subcommands are:
 
+- init : create a project
 - build : build the project
```
Comment (bguedas) [inline ApplicationDeveloperGuide/mmm.rst:436]:
```suggestion
- ``build``: build the project
```

File: ApplicationDeveloperGuide/mmm.rst [hunk 4]
```diff
@@ -431,6 +432,7 @@ In order to use the MMM CLI for your project:
...
 - publish : publish the project
 - run : run the application on the simulator
```
Comment (bguedas) [inline ApplicationDeveloperGuide/mmm.rst:438]:
```suggestion
- ``run``: run the application on the simulator
```

**Review by bguedas — CHANGES_REQUESTED** [review]
I didn't find how to suggest changes on multiple lines. By the way, I've done two things: remove the space before colon and put command around double back quotes.

**Comment by tdelhomenie-microej** [comment]
I applied your changes, thanks.

**Pull request merged**